battan40/Tainted_Tomatillos
# Fix/refactor-clean-debug

### Is this a fix or a feature?
- fix

### What is the change or fix?
- Allows full functionality of the search bar
- Reinforces separate component file modularity
- Cleans data in utils file and imports cleaning functions across components
- Refactors for DRY use cases of javaScript
- Removes functions from returns and places them in a Utils folder
- Moves all API calls to a Utils folder
- Organizes data flow at top of files in order of need
- Removes unused Routes
- Removes unneeded click-handling
- Verifies that cypress tests continue to pass with `true green` passes

### How should this be tested?
- run npm start on project folder branch
- change view for broken CSS
- Check for data about the movie not populating
- As a user use the search bar to attempt to break the app

### Linked Issues: 
- closes #29 #31 #33 #35
